website/docs/d/is_bare_metal_server.markdown

```diff
@@ -111,7 +111,17 @@ In addition to all argument reference list, you can access the following attribu
 - `profile` - (String) The name for this bare metal server profile
 - `resource_group` - (String) resource group id of the bare metal server.
 - `resource_type` - (String) The type of resource referenced
-- `status` - (String) The status of the bare metal server [ **failed**, **pending**, **restarting**, **running**, **starting**, **stopped**, **stopping** ]
+- `status` - (String) The status of the bare metal server.
```

Comment:
Verify the below design with Harini if content team is fine, rest LGTM.

Reply:
Discussed, and she agreed.